src/discretization/fist.jl

```diff
@@ -50,8 +50,8 @@ function discretize(polyarea::PolyArea, ::FIST)
       𝒫 = Chain(points[inds])
       n = nvertices(𝒫)
       # 3. update 𝒬 near clipped ear
-      isear(𝒫, i)   && (𝒬 = 𝒬 ∪ [mod1(i,n)])
-      isear(𝒫, i+1) && (𝒬 = 𝒬 ∪ [mod1(i+1,n)])
+      update_adjacent_ear!(𝒬, 𝒫, i-1, n)
+      update_adjacent_ear!(𝒬, 𝒫, i, n)
```

Comment:
Can we replace these two calls by a single function call `update_adjacent_ears!(Q, P, i)` that takes care of both adjacent years? That way the logic is fully encapsulated in the function without having to keep track of a sequence of calls with changing indices.

Reply:
You could also use broadcasting on `[i-1, i]` or map, if you want a single function call, but that's mostly a matter of style.